clang/test/CodeGen/RISCV/attr-hw-shadow-stack.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s -fcf-protection=return | FileCheck %s
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s | FileCheck -check-prefix=NOSHADOWSTACK %s
+
+int foo(int *a) { return *a; }
+
+// SHADOWSTACK-NOT: attributes {{.*}}"hw-shadow-stack"{{.*}}
```

Comment:
```suggestion
// NOSHADOWSTACK-NOT: attributes {{.*}}"hw-shadow-stack"{{.*}}
```
Should be something like that?